kubernetes/gaffer/templates/api/deployment.yaml

```diff
@@ -18,6 +18,7 @@ metadata:
   name: {{ include "gaffer.fullname" . }}-api
   labels:
     {{- include "gaffer.labels" . | nindent 4 }}
+    creator: {{.Values.creator}}
```

Comment:
This isn't required? Just pass in the 'creator' label as part of `gaffer.labels`.

Reply:
Just realised that `gaffer.labels` is a template, not a value. Rather than having a specific value for a `creator` label, we should make it generic. So maybe add `.Values.labels` so that users can pass in any labels they like. We should probably do this across all the charts for consistency so might be worth doing separately in another PR.